### DISCUSSION TITLE
error when receiving movies on iphone from other devices

### DISCUSSION THREAD

**Discussion by KashinKoji6229** [localsend/localsend#1931] [Discussion (English, International)]
Hi guys,
i have an error when i want to sent a movie file on my iphone since other devices..... few weeks ago it is working !
i don't know what is the problem, maybe IOS 18 update...?
Please developpers solve that problem.
Best regards

**Comment by aiya000** [comment]
I may have same issue!!
Thank you.

In my case, we succeed fetching movies from iOS18 (iPhone15) to my Android15 (Google Pixel Pro 9 Fold), **but fetched movies are corrupted.**

If I try to view corrupted movies, depending on media player app, failed to view.
Also on my photo app (FShot), generating the thumbnails is failed.

Please fix this issue.
Thanks!

**Reply by KashinKoji6229** [reply]
Hi, im sorry but it is not the same problem as me.
When i want to send a film movie in my iphone 16 PM from my computer or my other phone galaxy S24 ultra, the transfer run normally but at the end i have an error message…

**Reply by aiya000** [reply]
I'm sorry!
Understood ‘:D‘

I'll send a new another issue.
Please don't mind my comments!

**Comment by Manojk729** [comment]
Having the same issue. sending bar completes then in end just writes error.
